test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpCodeFixVerifier`2.cs

```diff
@@ -21,7 +21,7 @@ namespace ILLink.RoslynAnalyzer.Tests
 	/// <typeparam name="TTest">The test implementation to use.</typeparam>
 	public partial class CSharpCodeFixVerifier<TAnalyzer, TCodeFix>
 		   where TAnalyzer : DiagnosticAnalyzer, new()
-		   where TCodeFix : CodeFixProvider, new()
+		   where TCodeFix : Microsoft.CodeAnalysis.CodeFixes.CodeFixProvider, new()
```

Comment:
Why?

Reply:
By inserting a resource file in ILLink.CodeFixProvider I needed to add it as part of the using statements, creating ambiguity on what is a CodeFixProvider